<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolAreaDialog</class>
 <widget class="QDialog" name="ToolAreaDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>195</width>
    <height>125</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Area Tool</string>
  </property>
  <widget class="QPushButton" name="close_btn">
   <property name="geometry">
    <rect>
     <x>108</x>
     <y>93</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Close dialog</string>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="AttribAreaWidget" name="areawidget" native="true">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>9</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Double clic to start tool</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start_btn">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>93</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Start tool</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>ressources/icon_point.png</normaloff>ressources/icon_point.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="msg_label">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>72</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AttribAreaWidget</class>
   <extends>QWidget</extends>
   <header>attribareawidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
